### ISSUE TITLE
Doesn't work on @sveltejs/kit versions `1.0.0-next.191` and higher

### ISSUE THREAD

**Issue by cor** [cibernox/svelte-intl-precompile#30]
On sveltekit version `1.0.0-next.190` it works fine, but starting from 191 it produces the following error:

```
Cannot find package '/Users/cor/Developer/project/node_modules/precompile-intl-runtime/' imported from /Users/cor/Developer/project/node_modules/svelte-intl-precompile/index.js
```

with `precompile-intl-runtime` version `"0.6.2"`

**Comment by cibernox** [comment]
Thanks for reporting, I haven't updated yet. I'll try to reproduce it tonight.

**Comment by kbsali** [comment]
same here

**Comment by cibernox** [comment]
I was able to reproduce it. It seems to be cause by some change in viteJS 2.7 handles dependencies for SSR

**Comment by cor** [comment]
Alright, would it be difficult to fix?

**Comment by sastan** [comment]
That was exactly my problem. Sorry. Didn't check this repo for related issues. Should be fixed with https://github.com/cibernox/precompile-intl-runtime/pull/25

**Comment by cibernox** [comment]
This should be fixed now in 0.6.4. Thanks @sastan for the fix downstream! I'm not sure why Vite 2.7 doesn't handle ESM modules in transitive dependencies anymore.
